modules/ROOT/pages/astream-custom-roles.adoc

```diff
@@ -3,14 +3,34 @@
 :page-aliases: docs@astra-streaming::astream-custom-roles.adoc
 :page-tag: astra-streaming,security,secure,pulsar
 
-Within *Organization Settings* > *User Management*, you can see the permissions for a specific role by hovering over the number in the *Roles* column of the table. This will show the permissions granted to the role.
+Apache Pulsar was created with support for multi-tenancy.
+A Pulsar cluster can have multiple tenants, and tenants can be spread across multiple Pulsar clusters.
+A tenant might represent a product line, a business unit, or an application.
+However you use a tenant, the tenant must be secure and isolated, so each tenant has its own security context, resource utilization, and isolation policies.
+A tenant can contain multiple *namespaces*. Namespaces administrate all *topics* within them. This structure is built into the Pulsar persistent topic URL:
+[source,shell]
+----
+persistent://tenant/namespace/topic
+----
 
-image::ROOT:astream-roles.png[]
+{product_name} is running Pulsar under the hood, but DataStax takes care of administration at the tenant level, so administration of `tenants` is blocked in `pulsar-admin`.
 
-If the default roles don't meet your requirements, you can use custom roles that meet your organizational needs.
+[TIP]
+====
+If you want to control your own tenants in Pulsar, check out https://www.datastax.com/products/luna-streaming[Luna Streaming], a production-ready, open-source distribution of Apache Pulsar.
+====
+
+You can still manage roles, users, and organizations in the {astra_ui}.
```

Comment:
Expand this statement to map Pulsar concepts to Astra concepts. This can help someone that might understand Pulsar feel more comfortable with the differences in Astra. This will be good context to lead one into the guides below.